src/Stripe.net/Entities/StripeFileUpload.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace Stripe
 {
     using System;
+
```

Comment:
curious about that one, is it a convention you want to start respecting? I don't think it was in any of the StyleCop rules

Reply:
It doesn't seem to conventional in C#, so I've reverted.